.travis.yml

```diff
@@ -2,7 +2,7 @@ sudo: true
 os: linux
 language: go
 go: [1.9.2]
-go_import_path: github.com/mysterium/node
+go_import_path: github.com/MysteriumNetwork/node
```

Comment:
I am not sure about upper cased MysteriumNetwork, apart that it looks bad its again common practice and "not all file systems are case sensitive." (source: https://talks.golang.org/2014/names.slide#16)

Reply:
If the file system is case insensitive it will support both paths for import, the upper case will break nothing here. 
And [here](https://github.com/golang/go/issues/6430#issuecomment-66085787) is a comment, that describes that path of the URL is case sensitive since go get use it, it should support it.

Reply:
A. I would go for `github.com/mysterium-network/node`
or
B. A. Rename our organization to `github.com/mysterium-network/node`
or
C. Event rename our repo to `github.com/mysterium-network/go-node`

Reply:
Working with such repo in case-insensitive env might produce code that will be interpreted in other OS'es as lower case thus braking work flow per-se.

Reply:
For my any of **A, B** or **C** is good.
I just don't like the way when we need to rename cloned repo to start a build.

Reply:
My vote is for `github.com/mysterium-network/node`.

Reply:
I like github.com/mysteriumnetwork/go-node

Reply:
I renamed organization. Go ahead with name `github.com/mysteriumnetwork/node`